<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[running TC1]]>
    </line>
    <line>
      <![CDATA[close app]]>
    </line>
    <line>
      <![CDATA[close browser]]>
    </line>
  </reporter-output>
  <suite started-at="2022-11-14T23:37:38 IST" name="Default suite" finished-at="2022-11-14T23:37:59 IST" duration-ms="20524">
    <groups>
    </groups>
    <test started-at="2022-11-14T23:37:38 IST" name="Default test" finished-at="2022-11-14T23:37:59 IST" duration-ms="20524">
      <class name="com.pomclass_with_annotation.TestClass">
        <test-method is-config="true" signature="openbrowse()[pri:0, instance:com.pomclass_with_annotation.TestClass@4dbb42b7]" started-at="2022-11-14T23:37:38 IST" name="openbrowse" finished-at="2022-11-14T23:37:54 IST" duration-ms="15500" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openbrowse -->
        <test-method is-config="true" signature="openapp()[pri:0, instance:com.pomclass_with_annotation.TestClass@4dbb42b7]" started-at="2022-11-14T23:37:54 IST" name="openapp" finished-at="2022-11-14T23:37:54 IST" duration-ms="215" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openapp -->
        <test-method signature="Testcase()[pri:0, instance:com.pomclass_with_annotation.TestClass@4dbb42b7]" started-at="2022-11-14T23:37:54 IST" name="Testcase" finished-at="2022-11-14T23:37:59 IST" duration-ms="4769" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[if both are different then TC fails expected [Enter Time-Track] but found [actiTIME - Enter Time-Track]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: if both are different then TC fails expected [Enter Time-Track] but found [actiTIME - Enter Time-Track]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at com.pomclass_with_annotation.TestClass.Testcase(TestClass.java:52)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[running TC1]]>
            </line>
          </reporter-output>
        </test-method> <!-- Testcase -->
        <test-method is-config="true" signature="closeapp()[pri:0, instance:com.pomclass_with_annotation.TestClass@4dbb42b7]" started-at="2022-11-14T23:37:59 IST" name="closeapp" finished-at="2022-11-14T23:37:59 IST" duration-ms="1" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[close app]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeapp -->
        <test-method is-config="true" signature="closebrowser()[pri:0, instance:com.pomclass_with_annotation.TestClass@4dbb42b7]" started-at="2022-11-14T23:37:59 IST" name="closebrowser" finished-at="2022-11-14T23:37:59 IST" duration-ms="0" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[close browser]]>
            </line>
          </reporter-output>
        </test-method> <!-- closebrowser -->
      </class> <!-- com.pomclass_with_annotation.TestClass -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
